code/modules/clothing/spacesuits/rig.dm

```diff
@@ -101,6 +151,7 @@
 	species_fit = list("Vox")
 	_color = "syndi"
 	armor = list(melee = 60, bullet = 50, laser = 30,energy = 15, bomb = 35, bio = 100, rad = 60)
+	action_button_name = "Toggle Helmet Camera" //This helmet does not have a light, but we'll do as if

```

Comment:
It's implied that this helmet has an "internal light", but its spot is taken over by the camera. Syndie RIG camera coding is wonky, I'm just providing a basic and general work-around

Plus, even if you fucked with the no_light variable, it would only disable the camera until restored